### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by SuprunArtem**
- [DEMO LINK](https://SuprunArtem.github.io/layout_catalog/)
- [TEST REPORT LINK](https://SuprunArtem.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,385 @@
...
+
+        <div class="main__card card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="Apple imac 27"
+          >
+          <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
```
Comment (loralevitska) [inline src/index.html:179]:
```suggestion
          <h1 class="card__title">
            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
          </h1>
```

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,385 @@
...
+
+          <div class="card__price">
+            <p class="card__price-name">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a
+            href="#buy"
+            class="card__buy-button"
+          >
+          Buy
```
Comment (loralevitska) [inline src/index.html:309]:
```suggestion
          >
            Buy
```

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:7]: It is a good practice to make all colors variable

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    margin: 0;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    color: $color-dark;
+    font-size: 16px;
+    line-height: 18px;
+    margin: 0;
+  }
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:66]: Fix everywhere according this rule from checklist
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
hi, good job!  need some improvements
Try to pass all tests. Pay attention to `reviews` indents and number font-weight
 image

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,386 @@
...
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+     href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- <h1>Catalog</h1> -->
```
Comment (loralevitska) [inline src/index.html:15]: Remove comments

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,386 @@
...
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png"
+        alt="logo moyo">
```
Comment (loralevitska) [inline src/index.html:19]:
```suggestion
        <img src="images/logo.png" alt="logo moyo">
```

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 16px;
+    margin-bottom: 24px;
+  }
+
+  &__review-number {
+    font-weight: 400;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:41]: Remove redundant default value everywhere

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+    text-transform: uppercase;
+    background: $color-background;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      background-color: #fff;
```
Comment (loralevitska) [inline src/styles/blocks/card.scss:84]: Repeatable code move to variables
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost perfect)
Let's improve your code

**Comment by SuprunArtem** [comment]
accidentally deleted the previous repository, can it be restored and is it worth it?

File: src/styles/blocks/card.scss (hunk 4 above)
Comment (loralevitska) [reply src/styles/blocks/card.scss:66]: not fixed 
image

File: src/styles/utils/normalize.scss [hunk 9]
```diff
@@ -0,0 +1,9 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+
```
Comment (loralevitska) [inline src/styles/utils/normalize.scss:4]: Remove empty line

**Review by loralevitska — CHANGES_REQUESTED** [review]
Hi, I see you haven't fixed previous comments, maybe you did not push your changes? If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review

File: src/styles/blocks/stars.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (loralevitska) [inline src/styles/blocks/stars.scss:9]: Combine with `background` property

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**